src/baremetal.ld

```diff
@@ -22,6 +23,12 @@ SECTIONS
 
   . = DEFINED(ROM_START) ? RAM_START : .;
   .data : { *(.data) }
+  .bss : {
+    . = ALIGN(16);
```

Comment:
Why is align necessary here?
Wouldn't bss symbol incorrectly point to misaligned address, while actual data starts aligned?

Reply:
I don't think this align is strictly necessary, I just copied the entire .bss section declaration from elsewhere. Looking at the disassembly, it seems like the section begins where the first object is placed inside it, so there shouldn't be a difference between whether you move a `.` outside of section declaration or inside of it.

Anyway, I'll drop the align statements for the sake of avoiding unexplained complexity.